Microsoft.Toolkit.Uwp.UI.Controls/RemoteDevicePicker/KindToImageConverter.cs

```diff
@@ -0,0 +1,59 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using Windows.UI.Xaml.Data;
+
+namespace Microsoft.Toolkit.Uwp.UI.Converters
+{
+    /// <summary>
+    /// Converter to convert Device Type to Icon
+    /// </summary>
+    public class KindToImageConverter : IValueConverter
+    {
+        /// <inheritdoc/>
+        public object Convert(object value, Type targetType, object parameter, string language)
+        {
+            string finalvalue = string.Empty;
+            switch ((string)value)
+            {
+                case "Desktop":
```

Comment:
RemoteSystem API return the kind of device via a string. We could use an enum, but it is a wasted effort and also a problem for maintaining because a new device category can be added at anytime.

https://docs.microsoft.com/en-us/uwp/api/windows.system.remotesystems.remotesystem.kind#Windows_System_RemoteSystems_RemoteSystem_Kind

Reply:
@Code-ScottLe avknaidu added an enum to represent this, so I was just saying we could use it here instead to not have to maintain the names in two places.

Reply:
That was *somewhere* on my note when I was testing last night... somewhere (left it home today). But you are right. I was thinking something like removing this enum all together and let the converter handle the unknown device kind with some default kind/string/image.